src/main/java/org/irods/util/ResourceProperty.java

```diff
@@ -0,0 +1,23 @@
+package org.irods.util;
+
+public enum ResourceProperty {
```

Comment:
Documentation?

Reply:
I was going to add documentation with the other PR #94 and leave the issue opened until that got merged, but I can go ahead and do it now.

Reply:
If this enum is new, let's go ahead and add the docs.

Had the enum already existed, then handling the documentation in the other PR would be more acceptable.

Reply:
Okok makes sense.